library/include/hipblaslt-ext-op.h

```diff
@@ -170,6 +170,58 @@ HIPBLASLT_EXPORT hipblasStatus_t hipblasltExtAMax(const hipDataType datatype,
                                                   uint32_t          m,
                                                   uint32_t          n,
                                                   hipStream_t       stream);
+
+/*! \ingroup library_module
+ *  \brief Perform absmax on given 2-D tensor with scale input. Generate one absmax value and scaled 2-D tensor output.
+ *
+ *  \details
+ *  This function computes amax on given 1D-tensor.
+ *
+ *  @param[in]
+ *  datatype Datatype of input/output tensor, currently support HIP_R_32F and HIP_R_16F only.
```

Comment:
datatype Datatype of input tensor

Reply:
Here is still comment as "Datatype of input/output tensor", but it is only for input tensor